### PR TITLE
fix `Parentheses are lost by our js formatter` + --beautify flag for the debugging (close #2573)

### LINKED ISSUE #2573: Processing error for the https://qa.xreach.app/ page
Steps to reproduce:
* open the https://qa.xreach.app/ page in the playground
* white page will be displayed instead of origin page
* open Chrome  DevTools and see the script error
image

### PR DESCRIPTION
**Pull request by LavrovArtem**

### CODE REVIEW

File: Gulpfile.js [hunk 1]
```diff
@@ -211,7 +213,7 @@ gulp.step('travis-saucelabs-qunit', () => {
 gulp.task('test-client-travis', gulp.series('build', 'travis-saucelabs-qunit'));
 
 gulp.step('http-playground-server', () => {
-    return runPlayground();
+    return runPlayground({ needBeautifyScripts });
```
Comment (miherlosev) [inline Gulpfile.js:216]: What about creating a separate gulp task with beatified script?

**Review by miherlosev — CHANGES_REQUESTED** [review]

File: Gulpfile.js (hunk 1 above)
Comment (LavrovArtem) [reply Gulpfile.js:216]: This flag is usable for all playground tasks . We'll create a lot of tasks if combine playground tasks with the beatify.

Comment (miherlosev) [reply Gulpfile.js:216]: Ok

**Review by miherlosev — APPROVED** [review]

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Review by Farfurix — APPROVED** [review]

**Pull request merged**